extensions/arc/images/extensions.svg

```diff
@@ -0,0 +1,18 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" viewBox="0 0 18 18">
```

Comment:
Where did you get this icon from?

Reply:
https://iconcloud.design/search/extensions/2f3648210-de64dd68d